vendor/github.com/docker/docker/vendor.conf

```diff
@@ -125,7 +125,7 @@ github.com/containerd/ttrpc 94dde388801693c54f88a6596f713b51a8b30b2d
 github.com/gogo/googleapis 08a7655d27152912db7aaf4f983275eaf8d128ef
 
 # cluster
-github.com/docker/swarmkit 8852e8840e30d69db0b39a4a3d6447362e17c64f
+github.com/docker/swarmkit cfa742c8abe6f8e922f6e4e920153c408e7d9c3b
```

Comment:
Not critical, but SwarmKit should also be bumped  in this PR, as the protobufs are used in the CLI (to show default values). I have another PR open that also brings in these changes, so if that's merged first, it's already taken care of (see https://github.com/docker/cli/pull/1225)

Reply:
Then Ideally I should go back to Moby update swarmkit vndr there and then update moby vndr here. I cant directly update swarmkit vndr here, right ?

Reply:
Oh; sorry. perhaps my comment was confusing. When I update the (e.g.) github.com/docker/docker in `/vendor.conf`, I also check if the new version of github.com/docker/docker is expecting more recent versions of other dependencies. I do so by checking the nested `vendor.conf` of the `github.com/docker/docker` dependency (this file :smile:).

I then check any dependency that was changed there and that's also used by docker/cli, and check if it's more recent than the one currently used; if so, I consider updating the dependency in docker/cli as well.

Reply:
will update now. In that way one PR will have all related information.